server/sandbox_run.go

```diff
@@ -48,6 +48,19 @@ func (s *Server) privilegedSandbox(req *pb.RunPodSandboxRequest) bool {
 	return false
 }
 
+// trustedSandbox returns true if the sandbox will run trusted workloads.
+func (s *Server) trustedSandbox(req *pb.RunPodSandboxRequest) bool {
+	kubeAnnotations := req.GetConfig().GetAnnotations()
+
+	trustedAnnotation, ok := kubeAnnotations[annotations.TrustedSandbox]
+	if !ok {
+		// A sandbox is trusted by default.
+		return true
+	}
+
+	return trustedAnnotation == "true"
```

Comment:
might want to use `trustedAnnotation` from https://github.com/kubernetes-incubator/cri-o/pull/555/files#diff-91bbeda7eb98a7adc57b9e47e2cf5c2bR35